README.md

```diff
@@ -67,13 +67,13 @@ end
 
 
   kind = case year
-         when 1850..1889 then "Blues"
-         when 1890..1909 then "Ragtime"
-         when 1910..1929 then "New Orleans Jazz"
-         when 1930..1939 then "Swing"
-         when 1940..1950 then "Bebop"
-         else "Jazz"
-         end
+  when 1850..1889 then "Blues"
+  when 1890..1909 then "Ragtime"
+  when 1910..1929 then "New Orleans Jazz"
+  when 1930..1939 then "Swing"
+  when 1940..1950 then "Bebop"
+  else "Jazz"
+  end
```

Comment:
Pretty much the reason to do the dropped line assignment is to keep the case indented as deep as when & keep rubocop happy. That's the only reason I've started using that style, as rubocop shouts at me when I use my current style.

If we're changing the style guide for this (:+1::100:) then I'll revert to writing my usual style, which matches Harry's change.